backend/product-browse.go

```diff
@@ -253,6 +256,40 @@ func handleSearchRequest(w http.ResponseWriter, r *http.Request, page Page) {
 	http.Redirect(w, r, route, http.StatusSeeOther)
 }
 
+func handleLastLoadMoreRequest(w http.ResponseWriter, r *http.Request) {
+	EnableCors(w, r)
+	SetContentTypeJson(w)
+	timesShowMoreClicked := r.URL.Query().Get("timesShowMoreClicked")
+	if timesShowMoreClicked == "1" {
+		w.WriteHeader(http.StatusNotFound)
+	}
+	productsFile, err := ioutil.ReadFile(buildShowMorePath(timesShowMoreClicked))
+	if err != nil {
+		w.WriteHeader(http.StatusNotFound)
+	}
+	var productsRoot JsonRoot
+	err = json.Unmarshal(productsFile, &productsRoot)
```

Comment:
Why have a marshall/unmarshall step? You could simply write the file contents to the response.

Reply:
Removed!